`Forms` : Update Doc (#616)
Esri/arcgis-maps-sdk-kotlin-toolkit: toolkit/featureforms/README.md

```
*Note* : Once the `validationErrorVisibility`  is set to `Visible`, changing it back to `Automatic` will have no effect since all the fields have now gained focus to show any errors.

### Permissions

The `FeatureForm` component requires the following permissions to function correctly. These permissions are added to the AndroidManifest.xml file of the library. 
But the calling app should request these permissions at runtime. If the permissions are not granted, the respective functionality will be limited.

- `Manifest.permission.CAMERA` : Required to use the camera for the `BarcodeScannerFormInput` input type and the `AttachmentsFormElement` to add attachments.

## Behavior

- The `FeatureForm` composable will display the form fields in the order they are defined in the `FeatureFormDefinition`.

> - The component supports the following [FormElement](https://developers.arcgis.com/kotlin/api-reference/arcgis-maps-kotlin/com.arcgismaps.mapping.featureforms/-form-element/index.html?query=sealed%20class%20FormElement) types as part of its configuration.

Reviewer: ```suggestion
- The component supports the following configurable [FormElement](https://developers.arcgis.com/kotlin/api-reference/arcgis-maps-kotlin/com.arcgismaps.mapping.featureforms/-form-element/index.html?query=sealed%20class%20FormElement) types as user input fields.
```
